ci/travis.rb

```diff
@@ -124,7 +124,7 @@ def adapter
 
   def rake(*tasks)
     tasks.each do |task|
-      cmd = "bundle exec rake #{task}"
+      cmd = "travis_retry bundle exec rake #{task}"
```

Comment:
I never saw this command fail for it to need travis_retry, is there any reason?

Reply:
I thought it might help with stalled builds, but it seems like that's not the case. Will remove this for now.